src/components/Markdown/Markdown.scss

```diff
@@ -30,14 +38,19 @@
   h4,
   h5,
   h6 {
-    display: flex;
-    align-items: center;
     font-family: $font-stack-heading;
     font-weight: 600;
     line-height: 1.4;
-    margin: 1.5em 0 0.25em;
     color: getColor(fiord);
     word-break: break-all;
+    margin-bottom: 0.25em;
```

Comment:
can we use `scroll-margin-top` instead of all of these? https://css-tricks.com/fixed-headers-and-jump-links-the-solution-is-scroll-margin-top/